README.md

```diff
@@ -0,0 +1,175 @@
+# Table of Contents
+
+  * [Discovering Members within EC2 VPC](#discovering-members-within-ec2-vpc)
+  * [Debugging](#debugging)
+  * [Hazelcast Performance on AWS](#hazelcast-performance-on-aws)
+    * [Selecting EC2 Instance Type](#selecting-ec2-instance-type)
+    * [Dealing with Network Latency](#dealing-with-network-latency)
+    * [Selecting Virtualization](#selecting-virtualization)
+
+## Discovering Members within EC2 VPC
+
+Hazelcast supports Eureka V1 discovery. It is useful when you do not want to provide or you cannot provide the list of 
+possible IP addresses.
+
+Please note that this document does not cover details for Eureka Server installation on AWS regarding DNS, IAM roles or 
+any other required service configurations.
+
+Eureka can either have its location hard-coded or it can be found using DNS. Using DNS is much more flexible, therefore 
+given example configurations assume available DNS resolution for Eureka server.
+
+### Configuring Eureka Discovery for Hazelcast Cluster Members
+
+- Add the *hazelcast-eurekast-one.jar* dependency to your project. 
+- Disable join over multicast, TCP/IP and AWS by setting the `enabled` attribute of the related tags to `false`.
+- Enable Discovery SPI by adding "hazelcast.discovery.enabled" property to your config.
+- Add *eureka-client.properties* file to working directory or use `eureka.client.props` dynamic property to define 
+property file path without `properties` extension.
+
+The following is an example declarative configuration.
+
+```xml
+ <hazelcast>
+   ...
+    <properties>
+        <property name="hazelcast.discovery.enabled">true</property>
```

Comment:
I think we can remove it from the docs now, as it was a temp property only (and as far as I know discovery is now enabled by default - we need to verify though :))